torchbenchmark/models/LearningToPaint/__init__.py

```diff
@@ -1,99 +1,99 @@
 import cv2
 import torch
+import random
 import numpy as np
-import sys
+from .baseline.DRL.evaluator import Evaluator
+from .baseline.utils.util import *
+from .baseline.DRL.ddpg import DDPG
+from .baseline.DRL.multi import fastenv
 
-import torch.nn as nn
-import torch.nn.functional as F
-import torch.optim as optim
-from .baseline.utils.tensorboard import TensorBoard
-from .baseline.Renderer.model import FCN
-from .baseline.Renderer.stroke_gen import *
 from ...util.model import BenchmarkModel
 from torchbenchmark.tasks import REINFORCEMENT_LEARNING
 
 from argparse import Namespace
 
 torch.manual_seed(1337)
 np.random.seed(1337)
-torch.backends.cudnn.deterministic = True
-torch.backends.cudnn.benchmark = False
+torch.backends.cudnn.deterministic = False
+torch.backends.cudnn.benchmark = True
```

Comment:
Maybe we should set this config globally instead of setting it for individual models repeatedly. But we can do it in a separate PR.

Reply:
Sounds good to me!